examples/mobile/UIComponents/components/buttons/contained/emphasis/medium-emphasis-list.html

```diff
@@ -17,33 +17,68 @@ <h1>
 			</h1>
 		</div>
 		<div class="ui-content">
-			<ul class="ui-listview">
-				<li class="li-has-right-btn">List 1
-					<button data-style="contained" data-inline="true">
-							Radius : 18dp
-					</button>
+			<ul class="ui-listview ui-content-area">
+				<li class="ui-li-divider">
+					<div class="ui-li-text">
+						<span class="ui-li-text-title">
+							List 1
+						</span>
+					</div>
+					<div class="ui-li-action">
+						<button data-style="contained" data-inline="true">
+								Radius : 18dp
```

Comment:
It would be good to remove unnecessary tab.

Reply:
Done.